Integrations/integration-jamf.yml

```diff
@@ -30,213 +30,164 @@ configuration:
-                        {to: 'Realname', from: 'realname'},
-                        {to: 'Room', from: 'room'},
-                    ]
-                }
-            ],
-        },
-        'test-module': {
-            url: 'computers',
-            method: 'GET',
-        },
-    };
+    import requests
+    from requests.auth import HTTPBasicAuth
+    import re
+    import os
 
-    function createContext(data, id) {
-        var createContextSingle = function(obj) {
-            var res = {};
-            var keys = Object.keys(obj);
-            keys.forEach(function(k) {
-                var values = k.split('-');
-                var current = res;
-                for (var j = 0; j<values.length - 1; j++) {
-                    if (!current[values[j]]) {
-                        current[values[j]] = {};
-                    }
-                    current = current[values[j]];
-                }
-                current[values[j]] = obj[k];
-            });
-            if (!res.ID && id) {
-                res.ID = id;
-            }
-            return res;
-        };
-        if (data instanceof Array) {
-            var res = [];
-            for (var j=0; j < data.length; j++) {
-                res.push(createContextSingle(data[j]));
-            }
-            return res;
-        }
-        return createContextSingle(data);
-    }
+    HEADERS = {"Accept":"application/json"}
 
-    function mapObjFunction(mapFields, filter) {
-        var transformSingleObj= function(obj) {
-            var res = {};
-            mapFields.forEach(function(f) {
-                result = dq(obj, f.from);
-                console.log(result)
-                if (result || result === false) {
-                    res[f.to] = result;
-                }
-            });
-            if (filter && !filter(res)) {
-                return undefined;
+    base_url = demisto.params()['server']
+    user = demisto.params()['credentials']['identifier']
+    password = demisto.params()['credentials']['password']
+    auth = HTTPBasicAuth(user,password)
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    def jamf_get_computers():
+        endpoint = "/JSSResource/computers"
+        try:
+            contents = requests.get(base_url + endpoint,auth=auth,headers=HEADERS).json()
+            ec = {}
+            ec["JAMF"] = [{"ID": item["id"], "Name":item["name"]} for item in contents['computers']]
+
+            demisto.results({ 'Type' : entryTypes['note'], 'ContentsFormat' : formats['json'], 'Contents' : contents, "EntryContext": ec,"HumanReadable": tableToMarkdown("Computers", ec["JAMF"])})
+        except Exception,e:
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : "Unable to complete command.  Please check credentials and/or proxy settings" })
```

Comment:
![image](https://user-images.githubusercontent.com/7270217/37559490-c24b8892-2a40-11e8-8eff-92a63436e596.png)
This error is misleading. You should show the real error exception to the user. 
I executed `!jamf-get-computers-match match="name=Shai-k"` for example, and I got `"Unable to complete command.  Please check credentials and/or proxy settings"`. After removing the `try except` I got 
```
File "<string>", line 462, in tableToMarkdown
 IndexError: list index out of range
```